app/views/presenters/_digitization_queue_item_status_form.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= simple_form_for(view.model, data: {remote: true, auto_submit: true}) do |f| %>
   <div class="input-group input-group-sm flex-nowrap">
     <%= f.collection_select :status,
-          Admin::DigitizationQueueItem::STATUSES.collect {|s| [s.humanize, s]}, :second, :first, {},
+          view.model.class::STATUSES.collect {|s| [s.humanize, s]}, :second, :first, {},
```

Comment:
`view.model.class::STATUSES` is a bit weird ruby syntax, I"m a bit surprised it works! Although it is clever. 

Another way to do this, would be just make a `def available_statuses` instance method on each model, and then you can just call `view.model.available_statuses`. 

Both ways, rely on both classes implementing a certain convention. But this one seems a bit clearer to me.

Reply:
Good call. Done.